community/captains-log/index.md

```diff
@@ -26,52 +25,54 @@ Our community began as a partnership between Netflix & Google, eager to share th
 
 
 
-### GitHub Events in Spinnaker Repositories
+### GitHub events in Spinnaker repositories
 
-After steadily building momentum since its first open-source release in 2015, Spinnaker activity takes off:
+After steadily building momentum since its first OSS release in 2015, Spinnaker activity takes off. In December 2019, significant events within a week achieved a recent high of more than 4 times the weekly average so far:
 {%
   include
   figure
   image_path="./github_events.png"
 %}
 
-### Contributions Per Company
+### Contributions per company
 
-Building from a committed base of key organizations like Netflix, Google, Armory, OpsMx, and Amazon, 2019 saw significantly more contributions from end-user companies and new stakeholders:
+Building from a committed base of key organizations, 2019 saw a rise in contributions from end-user companies and new stakeholders like Armory, OpsMx, and Amazon. The average number of contributions from companies besides Netflix and Google in H2 2019 was more than 2.5 times the average from December 2015 to today:
 {%
   include
   figure
   image_path="./company_contributions.png"
 %}
 
-### Activity By Repository
+### Activity by repository
 
-The Spinnaker ecosystem currently includes 44 repositories, including the microservices that deliver its core functionality and interface with deployment targets such as AWS, GCP, and Kubernetes. Also included: cleanup and monitoring tools, documentation, governance, community resources, and much more:
+The Spinnaker ecosystem currently includes 44 repositories, including the microservices that deliver its core functionality and interface with deployment targets such as AWS, GCP, and Kubernetes. Added as the project has diversified: cleanup and monitoring tools, documentation, governance, community resources, and much more:
 {%
   include
   figure
   image_path="./activity_by_repo.png"
 %}
 
-### Companies and Developers Contributing Each Week
+### Companies and developers contributing each week
 
-Spinnaker has seen steadily increasing engagement from companies, and spikes of developer activity around community initiatives:
+Spinnaker has seen steadily increasing engagement from companies, and spikes of developer activity around community initiatives. The average weekly number of contributing developers was double the prior average in 2019, with up to 21 companies and 124 developers contributing each week:
 {%
   include
   figure
   image_path="./company_dev_count.png"
 %}
 
-### Myriad Pull Request Contributors
+### Myriad pull request contributors
 
-Unique authors contribute pull requests to Spinnaker repositories each week, building the project collaboratively over time.
+Unique authors contribute pull requests to Spinnaker repositories each week, building the project collaboratively over time. 314 developers submitted PRs in 2019, compared to 282 in 2018, and 194 in 2017:
```

Comment:
```suggestion
Unique authors contribute pull requests to Spinnaker repositories each week, building the project collaboratively over time. In 2019, 314 developers submitted PRs, compared to 282 in 2018, and 194 in 2017:
```

Reply:
Suggesting that change just fto avoid starting a sentence with a numeral.

Reply:
good call